_js/index.js

```diff
@@ -1,3 +1,5 @@
+var document = window.document;
```

Comment:
Is this needed? It doesn't seem to be used.

Reply:
This was old code, this is now removed.